### ISSUE TITLE
Fix DENYLIST format

### ISSUE THREAD

**Issue by chantra** [kernel-patches/vmtest#169]
We started to use 3-column format in https://github.com/kernel-patches/bpf/blob/bpf-next/tools/testing/selftests/bpf/DENYLIST.s390x, where the 3rd column identify where the problem is stemming from.

It would be useful to have the format
1) captured as a header of the file
2) used in the existing files so the next person that updates the file know what to put in.

**Comment by anakryiko** [comment]
it's not 3-column format, it's a list of test specifiers and everything after # is a comment. Do you mean to enforce some particular format of a comment or what is this about?

**Comment by chantra** [comment]
> it's not 3-column format, it's a list of test specifiers and everything after # is a comment. 
Yeah, I think it is a loose-3-column-format. @danielocfb commented about this in the PR. Is that what you meant (test<spaces># comment<spaces>failing_category)?

**Comment by danielocfb** [comment]
> > it's not 3-column format, it's a list of test specifiers and everything after # is a comment.
> > Yeah, I think it is a loose-3-column-format. @danielocfb commented about this in the PR. Is that what you meant (test# commentfailing_category)?

Yes, what we do in other deny lists. https://github.com/kernel-patches/bpf/blob/bpf-next/tools/testing/selftests/bpf/README.rst#denylisting-tests
